### PR TITLE
Update plugin information

### PR DESCRIPTION
**Pull request by mhmohona**
## Explanation

Updated existing plugin documentation.

## Related issue
#85 

## What type of PR is this
/kind documentation

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/contribute/source/plugin/rusttls.md [hunk 1]
```diff
@@ -2,9 +2,71 @@
 sidebar_position: 8
 ---
 
-# Build with Rusttls Plugin
+# Build with Rusttls Plug-in
```
Comment (alabulei1) [inline docs/contribute/source/plugin/rusttls.md:5]: May I ask where is the source for this doc?

**Comment by alabulei1** [comment]
Nice catch! Thanks for the PR. I have left some comments. Please take a look.

File: docs/contribute/source/plugin/rusttls.md (hunk 1 above)
Comment (mhmohona) [reply docs/contribute/source/plugin/rusttls.md:5]: This readme file - https://github.com/WasmEdge/WasmEdge/blob/master/plugins/wasm_bpf/README.md

**Review by alabulei1 — APPROVED** [review]

**Pull request merged**